src/seurat/workflows/multi_sample.nf

```diff
@@ -0,0 +1,94 @@
+nextflow.enable.dsl=2
+
+////////////////////////////////////////////////////////
+//  Import sub-workflows/processes from the utils module:
+include {
+    PUBLISH;
+} from '../../utils/workflows/utils.nf' params(params)
+include {
+    FILE_CONVERTER as FILE_CONVERTER_TO_SCOPE;
+} from '../../utils/workflows/fileConverter.nf' params(params)
+include {
+    UTILS__GENERATE_WORKFLOW_CONFIG_REPORT;
+} from '../../utils/processes/reports.nf' params(params)
+
+////////////////////////////////////////////////////////
+//  Import sub-workflows/processes from the tool module:
+include {
+    FILTER;
+} from './filter.nf' params(params)
+include {
+    MERGE;
+} from './merge.nf' params(params)
+include {
+    NORMALIZE;
+    NORMALIZE_SCALE_SCT;
+} from './normalize.nf' params(params)
+include {
+    HVG_SELECTION;
+} from './hvg_selection.nf' params(params)
+include {
+    DIM_REDUCTION_PCA;
+} from './dim_reduction_pca.nf' params(params)
+include {
+    DIM_REDUCTION_TSNE_UMAP;
+} from './dim_reduction.nf' params(params)
+include {
+    NEIGHBORHOOD_GRAPH;
+} from './neighborhood_graph.nf' params(params)
+include {
+    CLUSTERING
+} from './clustering.nf' params(params)
+include {
+    DIFFERENTIAL_GENE_EXPRESSION;
+} from './deg.nf' params(params)
+
+//////////////////////////////////////////////////////
+// Define the workflow
+
+workflow multi_sample {
+    take:
+        data
+    
+    main:
+        filtered = params.tools.seurat?.filter ? FILTER( data ).filtered : data
+
+        MERGE( 
+            filtered.map { it -> it[1] }.toList()
+        )
+        
+        // SCT workflow not supported in multiple sample, since this can have a negative influence on batch effects
+        NORMALIZE( MERGE.out ) | HVG_SELECTION
```

Comment:
In this case it would be nice to catch it and report it to the user as a warning if they've specified the parameter, rather than just doing the Seurat normalization without informing them.